docs_guide/sphinx_guide/how_to_build_docs.rst

```diff
@@ -0,0 +1,103 @@
+##################################
+How to build documentation locally
+##################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to build the documentation of your project in a local machine.
+
+Pre-requisites
+==============
+
+This guide assumes your project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+:doc:`sphinx-quickstart <sphinx:man/sphinx-quickstart>`.
+
+Install package and dependencies
+================================
+
+Before trying to build your documentation, make sure you have installed:
+
+* The Qiskit package you want to build, that in this guide will be called ``qiskit_x``.
+* Other required packages from the corresponding ``requirements-dev.txt``.
+
+Install the Qiskit package
+--------------------------
+
+Once you have cloned the ``qiskit-x`` repository with ``git clone https://github.com/Qiskit/qiskit-x.git``, you can install the ``qiskit_x`` package from source by going to the root directory of your local copy and running this on your console:
```

Comment:
Should we change the link here like we did in the tutorial to remove explicit references to [the Qiskit GitHub organization](https://github.com/Qiskit)?

What I mean is something like this:

`https://github.com/Qiskit/qiskit-x.git` $\rightarrow$ `https://github.com/X/qiskit-x.git`

Another option is to do the same as the tutorial and stick to `https://github.com/hal/qiskit-hal-provider.git` (and change `qiskit_x` or `qiskit-x` to `qiskit-hal-provider`)

Reply:
Changed to `git clone https://github.com/<your user/organization name>/qiskit-x.git`